packages/yourloops/components/dialogs/pdf-print-options.tsx

```diff
@@ -118,7 +123,8 @@ function getDatesFromPreset(preset: Presets, minDate: Dayjs, maxDate: Dayjs) {
   if (start.isBefore(minDate)) {
     start = minDate;
   }
-  return { start: start.format("YYYY-MM-DD"), end, preset };
+  const outputFormat = "pdf" as OutputFormat;
```

Comment:
```suggestion
  const outputFormat = "pdf" as OutputFormat;
```
```suggestion
  const outputFormat : OutputFormat = "pdf";
```
This is better at this guarantee that "pdf" is indeed an OutputFormat instead of casting him into one.